app.rb

```diff
@@ -16,16 +16,17 @@ def escape(memo)
   end
 end
 
-def memo_specified_by_id(params_id)
-  @connected_db.exec_params('SELECT * FROM memos WHERE id = $1;', [params_id])
+def find(params_id)
+  memo = @connected_db.exec_params('SELECT * FROM memos WHERE id = $1;', [params_id])
+  memo[0]
 end
 
 get '/' do
   redirect '/memos'
 end
 
 get '/memos' do
-  @memos = @connected_db.exec('SELECT * FROM memos')
+  @memos = @connected_db.exec('SELECT * FROM memos ORDER BY id DESC')
```

Comment:
```suggestion
  @memos = @connected_db.exec('SELECT * FROM memos ORDER BY id DESC;')
```

SQLにセミコロンを付けるのか付けないのか、統一した方がいいかも。
ちなみに僕は付けない派です（付けなくても動作に違いがないなら）。

Reply:
> SQLにセミコロンを付けるのか付けないのか、統一した方がいいかも。
ちなみに僕は付けない派です（付けなくても動作に違いがないなら）。

了解しました。
確かにその通りですよね、、
コードの書き方は統一するのが好ましいですし、、🙇‍♂️